cmd/heartbeats/main.go

```diff
@@ -122,7 +134,28 @@ func main() {
 		log.Fatalf("Failed to initialize tracing: %v", err)
 	}
 	defer tracer.Shutdown(ctx)
-	c, err := client.NewClientHTTP([]cehttp.Option{cloudevents.WithTarget(sink)}, nil)
+
+	opts := make([]cehttp.Option, 0)
+	opts = append(opts, cloudevents.WithTarget(sink))
+
+	if cacerts != "" && eventingtls.IsHttpsSink(sink) {
+		clientConfig := eventingtls.NewDefaultClientConfig()
+		clientConfig.CACerts = &cacerts
+
+		httpTransport := http.DefaultTransport.(*http.Transport).Clone()
+		httpTransport.TLSClientConfig, err = eventingtls.GetTLSClientConfig(clientConfig)
+		if err != nil {
+			log.Printf("Failed to get TLS Client Config: %v", err)
```

Comment:
I'd fail completely here instead of potentially getting errors during the connection or even work expecting the use of HTTPS and getting HTTP
```suggestion
			log.Fatalf("Failed to get TLS Client Config: %v", err)
```